gen/priv/eth_dev.h

```diff
@@ -0,0 +1,15 @@
+#ifndef TRAFFIC_GENERATOR_ETH_DEV_H
+#define TRAFFIC_GENERATOR_ETH_DEV_H
+
+#include "../../precompiled.h"
+
+#define VALID_RSS_HF 0x38d34
+
+#define RTE_RX_DESC_DEFAULT 1024
+#define RTE_TX_DESC_DEFAULT 1024
```

Comment:
In general it's better to put defines in the source files, if they are not needed in other files.
Also it's good practice to prefix the defines in given application/library/etc with a prefix which is specific for this application/library. You shouldn't use the `RTE_` prefix because it's for DPDK defines and this way you define may clash with some DPDK one.
Last, you don't need to setup RSS for the traffic generator because you are going to work with single queue only.
I mean, it's not wrong but it's not needed.